### ISSUE TITLE
Feature: Allow multiline dependent recipes

### ISSUE THREAD

**Issue by nyurik** [casey/just#1761]
If a recipe has many parameters, it becomes increasingly hard to manage a long list of them in one line. Ideally, it should be possible to have them use multiple lines with a backslash:

```justfile
task param: \
        (subtask param "abc-1" "xyz") \
        (subtask param "abc-2" "xyz") \
        (subtask param "abc-3" "xyz") 

subtask param1 param2 param3:
    echo "my super subtask {{ param1 }} {{ param2 }} {{ param3 }}"
```

## Workarounds
I am aware that I can do it with a nested call to just, but that is undesirable because the parameters have to undergo encoding/decoding, resulting in subtle bugs and hard to read code:

```justfile
task param:
    # param here must be encoded back into shell-compatible way,
    # wrapped with quotes, and hope that it does not break
    {{ just_executable() }} subtask "{{ param }}" "abc-1" "xyz"
    {{ just_executable() }} subtask "{{ param }}" "abc-2" "xyz"
    {{ just_executable() }} subtask "{{ param }}" "abc-3" "xyz"

subtask param1 param2 param3:
    echo "my super subtask {{ param1 }} {{ param2 }} {{ param3 }}"
```

**Comment by laniakea64** [comment]
> Ideally, it should be possible to have them use multiple lines with a backslash:

The exact syntax you're proposing [is already implemented as of `just` 1.15.0](https://just.systems/man/en/chapter_47.html#outside-recipe-bodies).

**Comment by nyurik** [comment]
@laniakea64 @casey thank you for the quick reply. The reason I thought it was not implemented is due to to IntelliJ Justfile plugin:

![image](https://github.com/casey/just/assets/1641515/7f043661-594a-4f2a-9334-56da5bdc0d60)
